lib/ex_aws/timestream/write/measure_value.ex

```diff
@@ -0,0 +1,23 @@
+defmodule ExAws.Timestream.Write.MeasureValue do
+  alias __MODULE__
+
+  @moduledoc """
+  Measure Value for the time-series data point.
+  https://docs.aws.amazon.com/timestream/latest/developerguide/API_MeasureValue.html
+  """
+
+  defstruct name: [],
+            value: nil,
+            type: nil
+
+  @type measure_value :: %MeasureValue{}
+
+  @doc "Create a new MeasureValue struct"
+  @type new_opts :: %{
+    name: binary,
+    value: binary,
+    type: binary,
+  }
```

Comment:
Is this type needed? It looks like maybe you were going to use it for a `new/1` and then went with `new/3` instead.

Reply:
Yep, you're right, will update.